yt_dlp/postprocessor/sponsorblock.py

```diff
@@ -0,0 +1,96 @@
+import json
+import re
+from hashlib import sha256
+
+from .ffmpeg import FFmpegPostProcessor
+from ..compat import compat_urllib_parse_urlencode, compat_HTTPError
+from ..utils import PostProcessingError, sanitized_Request
+
+
+class SponsorBlockPP(FFmpegPostProcessor):
+
+    EXTRACTORS = {
+        'Youtube': 'YouTube',
+    }
+    CATEGORIES = {
+        'sponsor': 'Sponsor',
+        'intro': 'Intermission/Intro Animation',
+        'outro': 'Endcards/Credits',
+        'selfpromo': 'Unpaid/Self Promotion',
```

Comment:
IMHO, old names were better.

Reply:
I personally don't care either way, but these are the actual names as documented here: https://wiki.sponsor.ajay.app/index.php/Segment_Categories